README.md

```diff
@@ -1 +1,43 @@
-# enumerables
\ No newline at end of file
+# Enumerables
+
+> Building my own description of methods.
+
+
+
+## Built With
+
+- Ruby
+
+
+## Live Demo
+
+[Live Demo Link](N/A)
+
+
+### Usage
+
+- open enumerable.rb file
+
+
+

```

Comment:
- Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.